requirements.txt

```diff
@@ -59,6 +59,7 @@ shapely>=1.3.0
 sphinx>=3
 # torch 1.7+ required for typing
 torch>=1.7
+torchaudio
```

Comment:
What's the minimum version that works? Also, don't forget to add to `docs/requirements.txt`

Reply:
Removed torchaudio dependency.